### PR TITLE
update help example

### PR DESCRIPTION
**Pull request by jonathanlukas**
## Description

This PR adds the result of `zbctl --help` from `8.5.6` which contains some new flags.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/apis-tools/cli-client/index.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ zbctl [options] [command]
...
 
 ```
-zbctl is a command line interface designed to create and read resources inside the Zeebe broker.
-It is designed for regular maintenance jobs, such as:
-        * Deploying processes
-        * Creating jobs and process instances
-        * Activating, completing, or failing jobs
-        * Updating variables and retries
-        * Viewing cluster status
+zbctl is a command line interface designed to create and read resources inside zeebe broker.
```
Comment (christinaausley) [inline docs/apis-tools/cli-client/index.md:42]: Do we know why these changes were reverted @jonathanlukas?

Comment (jonathanlukas) [reply docs/apis-tools/cli-client/index.md:42]: No idea actually, but it seems to come from my commit...

**Review by christinaausley — APPROVED** [review]
